### PR TITLE
Create post apoc price guide

### PR DESCRIPTION
**Pull request by Brian-Otten**
#### Summary
```SUMMARY: Infrastructure "Adds guide to pricing items to doc folder"```

#### Purpose of change
There was no guideline for giving items a sensible post_apoc price

#### Describe the solution
Add a guide to pricing items to /doc

#### Additional context
I hope people will find the guideline useful and concise for quickly adding prices when pushing new content. I'm open to feedback on how to improve the document.

### CODE REVIEW

**Comment by Xenomorph-III** [comment]
Well, there goes the days of selling $90 RAM sticks in exchange for every valuable item in stock.

File: doc/POSTAPOC_PRICE_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# How to give items a sensible postapoc_price value
+
+### Pricing philosophy
+Prices were based on the philosophy that there is a decent portion of humanity unwilling to regularly brave the risks of city scavenging, that are always on the lookout for supplies to stay alive. This mean food and medicine are relatively quite valuable compared to the goods you would use to acquire them.
+
+The standard unit of currency, the free merchants note, is priced to be the equivalent to one piece of meat jerky. (250 cents)
+One piece of meat jerky will stay for 24 days, a decent time, and contains 347 calories.
```
[the remark this answers is not in the document]
Comment (Brian-Otten) [reply doc/POSTAPOC_PRICE_GUIDE.md]: Thank you, missed that. Fresh seems fine.

File: doc/POSTAPOC_PRICE_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    "name": { "str": "aluminum bat" },
+    "description": "An aluminum baseball bat, lighter than a wooden bat and a little easier to swing as a result.",
+    "price_postapoc": 1250,
+  },
+
+#### Food pricing
+Food is priced mostly for a combination of calories, how preservable they are, and how nutritious they are.
+Meat jerky is 250
+Food that has more calories, is canned, and is very nutritious might be 500
+Food that las less calories, goes bad quickly, and is junk food, might be 50
```
Comment (Zireael07) [inline doc/POSTAPOC_PRICE_GUIDE.md:87]: "has"

File: doc/POSTAPOC_PRICE_GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
```
Comment (kevingranade) [inline doc/POSTAPOC_PRICE_GUIDE.md:87]:
```suggestion
Food that has less calories, goes bad quickly, and is junk food, might be 50
```

**Pull request merged**